lib/cucumber/multiline_argument/data_table.rb

```diff
@@ -161,7 +161,7 @@ def hashes
       def symbolic_hashes
         @symbolic_hashes ||=
           hashes.map do |string_hash|
-            Hash[string_hash.map { |a, b| [symbolize_key(a), b] }]
+            string_hash.transform_keys { |a| symbolize_key(a) }
```

Comment:
Are we confident that `string_hash` is always a hash?

Reply:
Would the code do something else as it would have before?

Reply:
Yes. Previously this co-erced the Hash. This is technically a breaking change (if the previous input "could" be a non-hash)

Reply:
I've checked.
`hashes` will always be an array of hashes.
It uses the private method `build_hashes` which ends with `.map(&:to_hash)`